backend/routers/filer.py

```diff
@@ -300,12 +300,7 @@ async def rollback_filer(cik: str, password: str):
 @cache(24)
 @router.get("/search", tags=["filers"], status_code=200)
 async def search_filers(q: str, limit: int = 4):
-    options = {"limit": limit, "filter": "thirteen_f = true"}
-    hits = search_companies(q, options)
-
-    results = []
-    for result in hits:
-        results.append(result)

```

Comment:
I removed this because it doesn't look to be doing anything, `results` is never used. Or maybe I'm missing something?